importer/management/commands/make_documents_list.py

```diff
@@ -172,12 +172,13 @@ def handle(self, *args, **options):
 
             publication.body = introduction
             publication.documents = json.dumps(new_stream_value)
+            first_published_at = publication.first_published_at
+            last_published_at = publication.last_published_at
+            revision_created_at = publication.latest_revision_created_at
             rev = publication.save_revision()
-            publication.first_published_at = publication.first_published_at
-            publication.last_published_at = publication.last_published_at
-            publication.latest_revision_created_at = (
-                publication.latest_revision_created_at
-            )
+            publication.first_published_at = first_published_at
+            publication.last_published_at = last_published_at
+            publication.latest_revision_created_at = revision_created_at
             publication.save()
             rev.publish()
```

Comment:
Isn't this going to be the offending line that actually updates the publication timestamp?

Reply:
Updated the title of the PR